cscs-checks/system/basic/file_system_check.py

```diff
@@ -0,0 +1,101 @@
+    def __init__(self):
+        self.descr = 'File system sanity test base'
+        # TODO: test from cn as well
+        self.valid_systems = ['daint:login', 'dom:login']
+        self.valid_prog_environs = ['builtin']
+        self.num_tasks = 1
+        self.num_tasks_per_node = 1
+        self.sanity_patterns = sn.assert_found(r'0', self.stdout)
+        self.perf_patterns = {
+            'real_time': sn.extractsingle(r'\nreal.+m(?P<real_time>\S+)s',
+                                     self.stderr, 'real_time', float)
+        }
+        self.postrun_cmds = ['echo $?']
+        self.tags = {'ops', 'diagnostic'}
+        self.maintainers = ['CB']
+
+
+@rfm.parameterized_test(['/scratch/snx1*'],
+                        ['/scratch/snx3*'])
+class FileSystemChangeDirCheck(FileSystemCommandCheck):
+    def __init__(self, variant):
+        super().__init__()
+        self.descr = 'Change directory to scratch test'
+        self.reference = {
+            'daint:login': {
+                'real_time': (0.1, None, 0.1, 's')
+            },
+            'dom:login': {
+                'real_time': (0.1, None, 0.1, 's')
+            }
+        }
+        self.executable = 'time cd'
+        self.executable_opts = [variant]
+
+
+@rfm.parameterized_test(['/scratch/snx1*'],
+                        ['/scratch/snx3*'])
+class FileSystemLsDirCheck(FileSystemCommandCheck):
+    def __init__(self, variant):
+        super().__init__()
+        self.descr = 'ls of directory in scratch test'
+        self.reference = {
+            'daint:login': {
+                'real_time': (0.1, None, 0.1, 's')
+            },
+            'dom:login': {
+                'real_time': (0.1, None, 0.1, 's')
+            }
+        }
+        self.executable = 'time ls'
+        self.executable_opts = [variant]
+
+                       
+@rfm.parameterized_test(['/project/csstaff/bignamic'],#TODO: can we parametrize the user?
+                        ['/users/bignamic'],
+                        ['/scratch/snx1*/bignamic'],
+                        ['/scratch/snx3*/bignamic'])#TODO: what is the correct way to do this?
+class FileSystemDuDirCheck(FileSystemCommandCheck):
+    def __init__(self, variant):
+        super().__init__()
+        self.descr = 'du of directory'
+
+        # TODO: is it possible to append a pattern?
+        self.perf_patterns = {
+            'real_time': sn.extractsingle(r'\nreal.+m(?P<real_time>\S+)s',
+                                          self.stderr, 'real_time', float),
+#            'size' : sn.extractsingle(r'(?P<size>\S+)\s+%s'%variant,
+#                                      self.stdout, 'size', float)
+            'size' : sn.extractsingle(r'(?P<size>\S+).+/bignamic',# TODO: this should be solved with parametrized user 
+                                      self.stdout, 'size', float)
+        }
+        
+        # TODO: system is not always relevant 
+        self.reference = {
+            '/project/csstaff/bignamic': {
```

Comment:
Does it make sense to have this?
```python
if user != 'jens...':
   self.valid_systems = []
```
We need to turn this off if the user is not our robot one. Or at least change the folder path if the user is the robot one.

Reply:
@victorusu which are the tests you want to turn off if the user is not jenscscs?